src/main/java/io/boomerang/service/refactor/TaskServiceImpl.java

```diff
@@ -498,7 +481,6 @@ public void endTask(InternalTaskResponse request) {
     List<String> keys = new LinkedList<>();
     keys.add(storeId);
 
-    workflowActivity = activityService.findWorkflowActivtyById(activity.getActivityId());
```

Comment:
I think this Entity is already retrieved at line 461, so no need to get it again.